Lib9c/BlockPolicy.cs

```diff
@@ -31,13 +34,21 @@ public class BlockPolicy : BlockPolicy<NCAction>
         {
             _minimumDifficulty = minimumDifficulty;
             _difficultyBoundDivisor = difficultyBoundDivisor;
+            _maximumTransactions = Math.Max(0, maximumTransactions);
```

Comment:
Why don't we just use [`uint`](https://docs.microsoft.com/en-us/dotnet/api/system.uint32?view=netcore-3.1) type?

Reply:
😲

Reply:
In .NET, I prefer `int` to `uint` in the interface of library because it isn't CLS-compliant. ([see also](https://stackoverflow.com/questions/1828575/why-should-i-write-cls-compliant-code)).

Reply:
😲